playbooks/scenarios/full/packet-flood.yml

```diff
@@ -37,7 +37,7 @@
     send_src_port: "{{ '%s' % range(2000, 10000) | random(seed=now|int + 2) }}"
     send_packet_count: 150
     min_received_packet_count: 99
-    max_received_packet_count: 150
+    max_received_packet_count: "{{ send_packet_count if send_protocol == 'UDP' else send_packet_count|int*3 }}"
```

Comment:
Can't do this. You will be masking an issue with dropping packets that currently is not occurring.
Please replace with either 150 or "{{ send_packet_count }}" which I actually like better.